docs/metrics/README.md

```diff
@@ -163,9 +163,9 @@ Here is the rule of thumb:
 
 > [!NOTE]
 > When reporting measurements with more than 8 tags, the API allocates memory on
-  the hot-path. You SHOULD try to keep the number of tags less than or equal to 8.
-  If you are exceeding this, check if you can model some of the tags as Resource,
-  as [shown here](#metrics-enrichment).
+  the hot code path. You SHOULD try to keep the number of tags less than or
```

Comment:
It's probably better to keep it as hot-path instead of "hot code path"?

Reply:
CoPilot recommends against "hot-path", a quick search shows that "hot path" and "hot code path" are widely used. e.g. https://www.hackterms.com/hot%20code